app/views/homepage/_promotion-slots.html.erb

```diff
@@ -2,119 +2,55 @@
-           data-track-dimension="<%= t('homepage.index.promo_boxes.title_1') %>">
-          <%= t('homepage.index.promo_boxes.title_1') %>
+      <div class="home-promo">
+        <a href="<%= item[:href] %>"
+          aria-hidden="true"
+          tabindex="-1"
+          data-track-category="homepageClicked"
+          data-track-action="promoImageLink"
+          data-track-label="<%= item[:href] %>"
+          data-track-value="1"
+          data-track-dimension-index="29"
+          data-track-dimension="<%= item[:image_src] %>">
+          <%= image_tag item[:image_src], {
+            alt: "",
+            class: "home-promo__image",
+            height: 407,
+            loading: "lazy",
+            width: 610,
+            sizes: "(max-width: 640px) 100vw, (max-width: 1020px) 33vw, 300px",
+            srcset: item[:srcset],
+          } %>
         </a>
-      </h3>
-      <p class="home-promo__text">
-        <%= t('homepage.index.promo_boxes.text_1') %>
-      </p>
-    </div>
 
-    <div class="govuk-grid-column-one-third">
-      <a href="https://www.nhs.uk/conditions/coronavirus-covid-19/coronavirus-vaccination/"
-         aria-hidden="true"
-         tabindex="-1"
-         data-track-category="homepageClicked"
-         data-track-action="promoImageLink"
-         data-track-label="https://www.nhs.uk/conditions/coronavirus-covid-19/coronavirus-vaccination/"
-         data-track-value="1"
-         data-track-dimension-index="29"
-         data-track-dimension="homepage/covid-19-vaccine-promo.png">
-        <%= image_tag "homepage/covid-19-vaccine-promo.png", {
-          alt: "",
-          class: "home-promo__image",
-          height: 407,
-          loading: "lazy",
-          width: 610,
-        } %>
-      </a>
-      <h3 class="home-promo__heading">
-        <a href="https://www.nhs.uk/conditions/coronavirus-covid-19/coronavirus-vaccination/"
-           class="govuk-link home-promo__link"
-           data-track-category="homepageClicked"
-           data-track-action="promoTextLink"
-           data-track-label="https://www.nhs.uk/conditions/coronavirus-covid-19/coronavirus-vaccination/"
-           data-track-value="1"
-           data-track-dimension-index="29"
-           data-track-dimension="<%= t('homepage.index.promo_boxes.title_2') %>">
-          <%= t('homepage.index.promo_boxes.title_2') %>
-        </a>
-      </h3>
-      <p class="home-promo__text">
-        <%= t('homepage.index.promo_boxes.text_2') %>
-      </p>
-    </div>
+        <h3 class="govuk-heading-m govuk-!-margin-bottom-2">
+          <a href="/travel-abroad"
+            class="govuk-link"
+            data-track-category="homepageClicked"
+            data-track-action="promoTextLink"
+            data-track-label="/travel-abroad"
+            data-track-value="1"
+            data-track-dimension-index="29"
+            data-track-dimension="<%= item[:title] %>">
+            <%= item[:title] %>
+          </a>
+        </h3>
```

Comment:
A heading component could be used here for the same effect:

```suggestion
        <%= render "govuk_publishing_components/components/heading", {
          font_size: "m",
          heading_level: 3,
          margin_bottom: 2,
          text: link_to(item[:title], "/travel-abroad", {
            class: "govuk-link",
            data: {
              track_category: "homepageClicked",
              track_action: "promoTextLink",
              track_label: "/travel-abroad",
              track_value: 1,
              track_dimension-index: 29,
              track_dimension: item[:title],
            }
          }),
        } %>
```

Reply:
Thanks, good one to know. I didn't realise I could pass `link_to` in the text property.